prometheus/src/util.ts

```diff
@@ -240,42 +252,77 @@ export function formatBytes(bytes: number) {
 /**
  * Calculates the time range based on the given interval.
  * @param {string} interval - The time interval (e.g., '10m', '1h', '24h', 'week').
- * @returns {Object} An object containing the 'from' timestamp, 'to' timestamp, and 'step' in seconds.
+ * @returns {Object} An object containing the 'from' timestamp and 'to' timestamp in seconds.
  */
-export function getTimeRange(interval: string): { from: number; to: number; step: number } {
+export function getTimeRange(interval: string): { from: number; to: number } {
```

Comment:
IMHO we can get resolution as an arg and calculate the step and return the step here itself, instead of first fetching the time range and then finding the step size. WDYT?